### ISSUE TITLE
SA1513 provides incorrect warning with interpolated string

### ISSUE THREAD

**Issue by MatthieuMEZIL** [DotNetAnalyzers/StyleCopAnalyzers#3442]
With the following code

```csharp
string suffix = $@"{(string.IsNullOrEmpty(extraSupport) ? string.Empty : $@"
        {extraSupport}")}
    }}
}}"
```

StyleCop 1.1.118 (not tested with the recent beta versions) generates a SA1513 on the closing curly bracket at the end of "{extraSupport}")}" which is incorrect
